lib/App/Config/Chronicle.pm

```diff
@@ -288,12 +307,9 @@ check and load updated settings from chronicle db
 sub check_for_update {
     my $self = shift;
 
-    # do fast cached check
-    my $now         = time;
-    my $prev_update = $self->_updated_at;
-    return if ($now - $prev_update < $self->refresh_interval);
+    return unless $self->_has_refresh_interval_passed();
+    $self->_updated_at(time);
```

Comment:
I'd probably indicate `Time::HiRes::time()` (I know its imported above, but a casual reading of here like in a diff would suggest otherwise.)